src/components/AppNavigationCaption/AppNavigationCaption.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+	<li class="app-navigation-caption--item">
+		{{ text }}
+	</li>
+</template>
+
+<script>
+
+export default {
+	name: 'AppNavigationCaption',
+	props: {
+		text: {
+			type: String,
+			required: true
+		}
+	}
+}
+</script>
+<style scoped>
+.app-navigation-caption--item {
+	font-weight: bold;
+	color: var(--color-text-maxcontrast);
+	line-height: 44px;
+	padding: 0 44px;
```

Comment:
```suggestion
	padding: 0 $clickable-area;
```

Though the padding seems a bit too big (88px?)

Reply:
This is how it was in `server`. We could change it though ...

Reply:
I assumed it was :stuck_out_tongue: 
No idea, it just did :boom: in my head when I read this.
I'm guessing this is for the left padding? To replace the icon?

If so `padding-left` should be enough?

Reply:
I will try it.

Reply:
All done.